specs/core/0_beacon-chain.md

```diff
@@ -691,11 +692,14 @@ def add_validator(validators: List[ValidatorRecord],
 ### Routine for removing a validator
 
 ```python
-def exit_validator(index, state, penalize, current_slot):
+def exit_validator(index, state, block, penalize, current_slot):
     validator = state.validators[index]
     validator.exit_slot = current_slot
     if penalize:
         validator.status = PENALIZED
+        whistleblower_xfer_amount = validator.deposit // SLASHING_WHISTLEBLOWER_REWARD_DENOMINATOR
+        validator.deposit -= whistleblower_xfer_amount
+        state.validators[get_proposer_index(state, block)].deposit += whistleblower_xfer_amount
```

Comment:
note: `get_proposer` is defined in #143
So we can update this line to:
```python
proposer = get_proposer(state, block)
proposer..deposit += whistleblower_xfer_amount
```

Reply:
Fixed.